scripts/all-targets-build

```diff
@@ -18,7 +18,10 @@ do
     for program in "${programs[@]}"
     do
         # Coremark and Dhrystone don't fit
-        if [ "${program}" == "dhrystone" -o "${program}" == "coremark" -o "${program}" == "example-freertos-blinky-systemview" ] ; then
+        if [ "${program}" == "dhrystone" -o \
+             "${program}" == "coremark" -o  \
+             "${program}" == "example-freertos-blinky-systemview" -o \
+             "${program}" == "example-buserror" ] ; then
```

Comment:
how come the size os large?

Reply:
I'm just adding `example-buserror` to an existing list.

The others haven't fit because the HiFive1 (and revb) only has 16k of DTIM.